misc/inforequests.md

```diff
@@ -0,0 +1,767 @@
+URL: /en/inforequests/create/<draft_id>/
+
+Conditions:
+* User is authenticated.
+* User has a verified e-mail address
+
+Instead of submitting the created inforequest directly, the user may decide to save it as a inforequest draft and submit it later. The user may edit the draft until he submits it. The obligee property does not have to be set while the inforequest is just a draft. The user may own multiple unfinished request drafts.
+
+### Delete a Inforequest Draft
+
+URL: /en/inforequests/delete-draft/<draft_id>/
+
+Conditions:
+* User is authenticated.
+* User is the request draft owner.
+
+## Views
+
+### ~~List of all Inforequests~~
+
+~~URL: /en/inforequests/...~~\
+~~Visibility: Public~~\
+~~The view contains a link to create a new inforequest.~~
+
+### List of Inforequests Created by the User
+URL: /en/inforequests/\
+Visibility: Authenticated user\
+The user may see his inforequest drafts besides his submitted and closed inforequests as well. The
+view contains a link to create a new inforequest.
+
+### Inforequest Detail
+URL: /en/inforequests/detail/<inforequest_id>/\
+Visibility: Public
+1. _For the applicant_:\
+   The applicant sees basic inforequest information, branch of actions in chronological order by
+   their effective date and his undecided e‑mails.\
+   The applicant may act on the inforequest if it's not closed, yet. He can:
+   - decide undecided e‑mails,
+   - add obligee actions based on obligee responses received by s‑mail,
+   - extend missed obligee deadlines,
+   - ~~resend undelivered e‑mail by s‑mail,~~
+   - send a clarification response,
+   - make an appeal.
+2. ~~For other users and the public:~~
+   ~~Other users and the public see only anonymized request information, branch of actions, and~~
+   ~~undecided e‑mails. If possible, content of actions and e‑mails should be anonymized as well.~~\
+   ~~Neither other users nor the public may act on the case.~~
+
+If the inforequest was advanced, details of all its descendant branches are shown in a tree
+structure as well.
+
+### Upload Attachment
+URL: /en/inforequests/attachments/\
+Visibility: Authenticated user\
+Uploaded files are attached to the user. The user may use uploaded files while composing an
+inforequest or an action. He may use only his uploaded files.
+
+### Download Attachment
+URL: /en/inforequests/attachments/<attachment_id>/\
+~~Visibility: Public~~\
+~~Anonymous user may download only files attached to actions connected to public inforequests.~~
+Authenticated user may also download files attached to him, to his inforequest drafts or to any
+e‑mails, actions or action drafts connected to his inforequests.
+
+
+## Administration
+
+### List of Unassigned Received E‑mails
+
+* May assign an unassigned e‑mail to a case
+
+### List of Undecided Raw E‑mails Assigned to Closed Inforequests
+
+### List of E‑mails Marked as Unrelated
+
+### List of E‑mails the Users didn't Know how to Decide
+* May decide the e‑mail
```

Comment:
Ked mame len takto zoznam prazdnych veci, tak to nedavaj ako napisny, lebo je to somarina mat kopec prazdnych sekcii. Pouzi odrazky ako som pisal minule. Cize:

```
* **List of Unassigned Received E‑mails**
  - May assign an unassigned e‑mail to a case
* **List of Undecided Raw E‑mails Assigned to Closed Inforequests**
...
```

Reply:
:ok: